src/settings/OrderTemplates/OrderTemplatesEditor/POLineVendorForm/POLineVendorForm.js

```diff
@@ -13,21 +14,29 @@ import {
   FieldVendorNote,
 } from '../../../../common/POLFields';
 
-const POLineVendorForm = () => {
+const POLineVendorForm = ({ change, dispatch }) => {
   return (
     <Row>
       <Col
         xs={3}
         data-col-order-template-vendor-number
       >
-        <FieldVendorRefNumber />
+        <FieldVendorRefNumber
+          change={change}
```

Comment:
I think you can remove all these passes of `change` and `dispatch`

Reply:
Removed all unnecessary prop